guides/source/active_record_migrations.md

```diff
@@ -78,8 +78,8 @@ end
 We define the change that we want to happen moving forward in time. Before this
 migration is run, there will be no table. After it is run, the table will exist.
 Active Record knows how to reverse this migration as well; if we roll this
-migration back, it will remove the table. You can read more about rolling back
-migrations [here](#rolling-back).
+migration back, it will remove the table. You can read more about [rolling back
+migrations](#rolling-back).
```

Comment:
The sentence reads a bit better if we remove the "You can", similar to the first change above.
```suggestion
migration back, it will remove the table. Read more about [rolling back
migrations](#rolling-back).
```
Or maybe:
```suggestion
migration back, it will remove the table. You can read more about rolling back migrations in
in the [Rolling Back](#rolling-back) section.
```

Reply:
Thanks, I'll add with the other link changes!